test/explicit-resource-management/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "explicit-resource-management",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "run": "npm run build && node out/main.js",
+    "build": "tsc",
+    "test": "npm run build && mocha --config .mocharc.json main.test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/chai": "^4.3.16",
+    "chai": "^4.4.1",
+    "mongodb": "../../mongodb-6.8.0.tgz",
```

Comment:
That's a good point.  We'll probably need to rename this file.  6.8.0 does include the current `version` from the package.json, so this will change after the next release.  

I'll rename the packed file to a stable filename and depend on that instead.